eng/testing/tests.targets

```diff
@@ -119,18 +119,18 @@
       -->
     <ItemGroup>
       <TestBinaries Include="$(OutDir)\*.*"/>
-      <AndroidTestRunnerBinaries Include="$(AndroidTestRunner)\*.*" />
-      <BclBinaries Include="$(NETCoreAppRuntimePackLibPath)\*.*"
-                   Exclude="$(NETCoreAppRuntimePackLibPath)\System.Runtime.WindowsRuntime.dll" />
-      <BclBinaries Include="$(NETCoreAppRuntimePackNativePath)\*.*" Exclude="$(NETCoreAppRuntimePackNativePath)\libmono.dylib" />
+      <AndroidTestRunnerBinaries Include="$(AndroidTestRunnerDir)*.*" />
+      <BclBinaries Include="$(RuntimePackLibDir)\*.*"
+                   Exclude="$(RuntimePackLibDir)\System.Runtime.WindowsRuntime.dll" />
```

Comment:
```suggestion
      <BclBinaries Include="$(RuntimePackLibDir)*.*"
                   Exclude="$(RuntimePackLibDir)System.Runtime.WindowsRuntime.dll" />
```

Reply:
I have another pending PR that touches these lines so I'll fold these cleanups into that one.